src/main/java/uk/gov/justice/hmpps/prison/service/transformers/OffenderTransformer.java

```diff
@@ -101,6 +103,18 @@ public InmateDetail transform(final OffenderBooking latestBooking) {
             .toList();
     }
 
+    private @NotNull List<OffenceHistoryDetail> getActiveConvictedOffences(Long rootOffenderId) {
+        return offenderChargeRepository.findChargesByRootOffenderId(rootOffenderId).stream()
+            .filter(oc -> "A".equals(oc.getChargeStatus()))
+            .filter(oc -> "A".equals(oc.getOffenderCourtCase().getCaseStatus().map(ReferenceCode::getCode).orElse(null)))
+            // According to SentenceRepositorySql.GET_OFFENCES_FOR_BOOKING this is to "Avoid dups from merges (from NART team)"
+            .filter(oc -> !"MERGE".equals(oc.getAuditModuleName()) || !"SYS".equals(oc.getCreateUserId()))
+            .sorted(Comparator.comparing(OffenderCharge::getId))
+            .map(offenderChargeTransformer::convert)
+            .filter(OffenceHistoryDetail::convicted)
+            .toList();
+    }
```

Comment:
This is the main change - only consider active charges and court cases when calculating the recall flag